lib/bootstrap.ts

```diff
@@ -37,6 +35,7 @@ export default function run(projectPkg?: any)  {
   debug('discovering commands from addons');
   let commands: { [key: string]: typeof Command } = {};
   let coreCommands: { [key: string]: typeof Command };
+  let cliCommands: { [key: string]: typeof Command };
```

Comment:
Can we call this `globalCommands` perhaps? CLI commands feels a tad unspecific - all the commands are CLI commands after all.